src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -42,6 +42,17 @@ public static class WorkerRuntimeLanguageHelper
             { WorkerRuntime.powershell, "powershell" }
         };
 
+        public static IDictionary<WorkerRuntime, string> WorkerToDisplayStrings => availableWorkersRuntime.Keys.ToDictionary(wr => wr, wr =>
```

Comment:
nit: because of the lambdas, I think the expression bodied member here is impacting readability a bit, can we just switch and have a body instead?

Reply:
Sounds good, will update